CMakeLists.txt

```diff
@@ -2093,7 +2093,7 @@ add_custom_command(
     COMMAND ${CMAKE_COMMAND}
         -DPROJECT_BINARY_DIR=${PROJECT_BINARY_DIR}
         -DAPPLICATION_SOURCE_DIR=${APPLICATION_SOURCE_DIR}
-        -DINTERFACE_INCLUDE_DIRECTORIES="$<JOIN:$<TARGET_PROPERTY:zephyr_interface,INTERFACE_INCLUDE_DIRECTORIES>,:>"
+        -DINTERFACE_INCLUDE_DIRECTORIES="$<JOIN:$<TARGET_PROPERTY:zephyr_interface,INTERFACE_INCLUDE_DIRECTORIES>,|>"
```

Comment:
CMake support `;`-separated list as argument, like:
```
cmake -DMY_LIST="first_item;second_item;third_item;...."
```

so why not pass this as a CMake list instead of going through another separator ?

Reply:
Fair!